nixos/modules/services/networking/charybdis.nix

```
@@ -81,9 +81,9 @@ in
         gid = config.ids.gids.ircd;
       };
 
-      systemd.tmpfiles.rules = [
-        "d ${cfg.statedir} - ${cfg.user} ${cfg.group} - -"
-      ];
+      systemd.tmpfiles.settings.${cfg.statedir}.d = {
```

Missing `10-charybdis`?

```suggestion
      systemd.tmpfiles.settings."10-charybdis".${cfg.statedir}.d = {
```

---

Yup, that's a slip :)